tests/test_api.py

```diff
@@ -160,7 +160,7 @@ def test_extract_references_from_url(pdf_files):
 
 def test_long_registrant_dois(pdf_files):
     """ DOIs with 5 digit registrant code """
-    r = extract_references_from_file(pdf_files[6])
+    r = extract_references_from_file(pdf_files[7])
```

Comment:
why do we need this change? 🤔

Reply:
Because when I added new pdf for testing it went to index 6 in `pdf_files` list, and file from index 6 was moved to 7 ;)